service/src/main/java/com/ericsson/eiffel/remrem/generate/controller/RemremGenerateController.java

```diff
@@ -159,6 +162,12 @@ public ResponseEntity<?> generate(final String msgProtocol, final String msgType
 
             if (inputData.isJsonArray()) {
                 JsonArray inputEventJsonArray = inputData.getAsJsonArray();
+
+                if (inputEventJsonArray.size() > maxSizeOfInputArray) {
+                    return createResponseEntity(HttpStatus.BAD_REQUEST, JSON_ERROR_STATUS,
+                            "The number of events in the input array is to high: " + inputEventJsonArray.size() + " > " + maxSizeOfInputArray + ", " +
```

Comment:
-> maxSizeOfInputArray + "; " + "you can ... it."

Reply:
Done